yacl/crypto/primitives/tpre/BUILD.bazel

```diff
@@ -0,0 +1,171 @@
+load("//bazel:yacl.bzl", "AES_COPT_FLAGS", "yacl_cc_binary", "yacl_cc_library", "yacl_cc_test")
+
+package(default_visibility = ["//visibility:public"])
+
+yacl_cc_library(
+    name = "kdf",
+    srcs = [
+        "kdf.cc",
+    ],
+    hdrs = [
+        "kdf.h",
+    ],
+    linkopts = ["-ldl"],
+    deps = [
+        "//yacl/crypto/base/hash:hash_utils"
+    ],
+    visibility = ["//visibility:public"],  
```

Comment:
You don't need to declare the visibility again, since you have include a default visibility setting